### DISCUSSION TITLE
osu！lazerFatal error in game map

### DISCUSSION THREAD

**Discussion by qwqDICEN** [ppy/osu#24180] [Q&A]
### Type

Performance

### Bug description

I download OSU! Lazer to my android phone.When I entered the game, the mistake happened.I tried reinstalling, but that didn't work.

My mobile phone model :redmi 10X Pro
Android version :12

### Screenshots or videos

![Screenshot_2023-07-11-01-17-09-437_sh ppy osulazer](https://github.com/ppy/osu/assets/139208851/394aa18c-304f-49d0-9bfc-4e2c2e247760)
![Screenshot_2023-07-11-01-22-25-279_sh ppy osulazer](https://github.com/ppy/osu/assets/139208851/10232f86-94f9-46ad-9420-d1ff7a6e0ab2)
![Screenshot_2023-07-11-01-22-52-752_sh ppy osulazer](https://github.com/ppy/osu/assets/139208851/0c9e7ee1-384e-45c3-9086-97e290199260)
![Screenshot_2023-07-11-01-22-31-825_sh ppy osulazer](https://github.com/ppy/osu/assets/139208851/ed82aea6-2ac3-48c3-b4ab-deb81db409d2)

### Version

2023.621.0

### Logs

[database.log](https://github.com/ppy/osu/files/12011618/database.log)
[network.log](https://github.com/ppy/osu/files/12011621/network.log)
[performance.log](https://github.com/ppy/osu/files/12011623/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12011624/runtime.log)

**Comment by Nine-Month** [comment]
Same.My problem occurs after not launching the game for a long time.
My mobile phone model :Redmi K30 Ultra
Android version :12
